tests/ui/resolve/local-shadows-glob-reexport.rs

```diff
@@ -0,0 +1,39 @@
+#![deny(local_binding_shadows_glob_reexport)]
```

Comment:
Warnings can be tested with
```
//~^ WARN message
```
so it's not necessary to deny lints to test them.

Reply:
I *think* the reason for this deny is that this lint is warn-by-default, and UI tests expect that compilation fails, so if only warnings are issued then doing `--bless` it complains about "error: ui test compiled successfully!".